weather-app-webpack/src/index.js

```diff
@@ -0,0 +1,62 @@
+import css from '../public/style.css';
+
+const script = document.querySelector('script');
+
+const wrapper = document.createElement('div');
+wrapper.classList.add('wrapper');
+document.body.insertBefore(wrapper, script);
+
+const main = document.createElement('div');
+main.classList.add('main');
+wrapper.appendChild(main);
+
+const mapEl = document.createElement('div');
+mapEl.setAttribute('id', 'map');
+main.appendChild(mapEl);
+
+const { form } = require('./searchForm');
```

Comment:
1) all _require, imports_ always **on top** of the file!
2) Why don't you use _imports/exports_ instead of _requires_?